packages/template-retail-react-app/app/pages/home/index.jsx

```diff
@@ -292,6 +295,8 @@ Home.getProps = async ({res, api}) => {
     return {productSearchResult}
 }
 
+Home.getLayout = getLayout
```

Comment:
Currently, implementing a `getLayout` in a page like homepage does not override the "global layout" found in _app component.

One common use case would be to override the global header and footer for certain pages (like Checkout having its own header, footer). It doesn't look like that is possible currently.

Reply:
I updated the example to allow overriding the default Layout defined in the `<App />` component.

An example can be seen in the Checkout pages.